CSharp/AccountManagement/AccountManagement.csproj

```diff
@@ -45,9 +45,9 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>packages\Microsoft.Azure.Common.2.1.0\lib\net45\Microsoft.Azure.Common.NetFramework.dll</HintPath>
     </Reference>
-    <Reference Include="Microsoft.Azure.Management.Batch, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>

```

Comment:
💡 Quick note here: Just wondering why the `specificversion` is removed? is it VS doing it for you without informing you : ) details as to how this works is detailed here: http://stackoverflow.com/questions/24022134/how-exactly-does-the-specific-version-property-of-an-assembly-reference-work-i